cvat/apps/dataset_manager/bindings.py

```diff
@@ -17,8 +17,8 @@
 import rq
 from attr import attrib, attrs
 from datumaro.components.dataset import Dataset
+from datumaro.components.media import ByteImage, Image, PointCloud
```

Comment:
Please check if using `import datumaro as dm` is reasonable here.

Reply:
Updated. But it doesn't work for `PointCloud`.